### PR TITLE
Add Singularity Cache Directory

### PR DESCRIPTION
**Pull request by apeltzer**

### CODE REVIEW

File: conf/cfc.config [hunk 1]
```diff
@@ -7,6 +7,7 @@ params {
 
 singularity {
   enabled = true
+  cacheDir = '/nfsmounts/containers'
```
Comment (apeltzer) [inline conf/cfc.config]:
```suggestion
  cacheDir = '/nfsmounts/container'
```

**Comment by drpatelh** [comment]
@apeltzer Something we discussed a while back and I havent got around to testing because I dont know if I am allowed to use the file paths on our cluster (for security reasons).

I thought it may be nice to use something like:
`cacheDir = "/path/to/local/dir/${workflow.manifest.name}/v${workflow.manifest.version}/"`

Where you would get separate folders for containers from different pipelines, and so they would be organised better. Putting all of the containers in a single directory may get messy especially when we move to modules so probably something we need to think about now?

If so, fancy testing it?

**Comment by apeltzer** [comment]
Wow @drpatelh ! Sure I fancy testing this 🎉

**Comment by drpatelh** [comment]
Thanks :100:

**Pull request merged**